ui/components/app/wallet-overview/eth-overview.js

```diff
@@ -250,6 +255,15 @@ const EthOverview = ({ className, showAddress }) => {
                   },
                 });
               }}
+              tooltipRender={
+                signingEnabled
+                  ? null
+                  : (contents) => (
+                      <Tooltip title={t('accountCannotSign')} position="bottom">
+                        {contents}
+                      </Tooltip>
+                    )
+              }
```

Comment:
Same here, this can also be reverted

Reply:
Reverted here [b427bcb](https://github.com/MetaMask/metamask-extension/pull/23178/commits/b427bcbbbbcda53f38507e7ae618f6e046551c8e)